dev/com.ibm.websphere.security.wim.base/src/com/ibm/websphere/security/wim/ConfigConstants.java

```diff
@@ -716,4 +716,25 @@
      * Custom property if set indicates to VMM that it needs encode certain characters while creating search expression.
      */
     String CONFIG_CUSTOM_PROP_USE_ENCODING_IN_SEARCH_EXPRESSION = "useEncodingInSearchExpression";
+
+    // String constants for bind authentication types
+    String CONFIG_PROP_BIND_AUTH_MECH = "bindAuthMechanism";
+    String CONFIG_BIND_AUTH_KRB5 = "GSSAPI"; // Upper case required for setting on Context.SECURITY_AUTHENTICATION
+    String MESSAGE_KERBEROS = " (Kerberos)";
+
+    /**
+     * List of supported bindAuthMechanism types. If a constant is added or removed,
+     * then this list should be updated too.
+     */
+    String[] CONFIG_BIND_AUTH_TYPES = {
+                                        CONFIG_AUTHENTICATION_TYPE_NONE,
+                                        CONFIG_AUTHENTICATION_TYPE_SIMPLE,
+                                        CONFIG_BIND_AUTH_KRB5
+    };
+
+    String CONFIG_PROP_KRB5_AUTH = "krb5Authentication";
+    String CONFIG_PROP_KRB5_PRINCIPAL = "krb5Principal";
+    String CONFIG_PROP_KRB5_TICKET_CACHE = "krb5TicketCache";
+    String CONFIG_PROP_KRB5_CONFIG = "configFile"; // must match the config on kerberos element
```

Comment:
Not sure we need CONFIG_PROP_KRB5_TICKET_CACHE or CONFIG_PROP_KRB5_CONFIG as that comes in via the kerberos configuration element that already exists. I assume it is a service we can use as a reference and directly access those values.

Reply:
I was debating keeping or leaving too. I removed, we can always add them back if we need them for something.